seedlet-blocks/block-templates/index.html

```diff
@@ -3,11 +3,21 @@
 <!-- /wp:group -->
 
 <!-- wp:group {"align":"full","className":"site-content","tagName":"main"} -->
-<div class="wp-block-group alignfull site-content"><div class="wp-block-group__inner-container"><!-- wp:query-loop -->
-<!-- wp:post-title /-->
+<div class="wp-block-group alignfull site-content"><div class="wp-block-group__inner-container">
+<!-- wp:query {"queryId":1,"query":{"perPage":10,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
+<!-- wp:query-loop -->
+<!-- wp:post-title {"isLink":true} /-->
 
 <!-- wp:post-content /-->
-<!-- /wp:query-loop --></div></div>
+
+<!-- wp:post-date {"fontSize":"tiny"} /-->
+
+<!-- wp:spacer {"height":60} -->
```

Comment:
Clever use of the spacer block inside the query block.

Reply:
The query block is basically just a dynamic wrapper — so anything you add in there will repeat for each post. It doesn't leave much padding between posts by default, so this helps with that.